examples/react-demo/src/app/page.tsx

```diff
@@ -79,15 +111,37 @@ export default function Home(): ReactElement {
     [sendTransaction],
   );
 
+  const sendMessage = useCallback(
+    (message: string) => {
+      const data = eth.abi.encodeFunctionCall(
+        contractAbiSendMessage as AbiFunctionFragment,
+        [message],
+      );
+
+      const transactionData = new TransactionBuilder({
+        to: CONTRACT_ADDRESS,
+        data,
+      });
+
+      void sendTransaction({
+        chainId: SEPOLIA_CHAIN_ID,
+        transaction:
+          "0x" + Buffer.from(transactionData.serialize()).toString("hex"),
```

Comment:
nit: https://docs.web3js.org/api/web3-utils/function/toHex

so we don't have to use Buffer polyfill